README.md

```diff
@@ -229,6 +232,45 @@ collections in your templates.
 <h1>Harry</h1>
 ```
 
+### Embedded engines
+
+Examples:
+
+```slim
+javascript:
+  console.log("Test javascript");
+
+css:
+  body {
+    color: black;
+  }
+
+elixir:
+  a = [1, 2, 3]
+  b = Enum.map(a, &(&1 + 1))
+
+eex:
+  Hello from <%= "eex" %>
+```
+
+You can define your own embedded engine in slime application config:
+
+```elixir
+# config.exs
+config :slime, :embedded_engines, %{
+  "markdown" => MyApp.MarkdownEngine
```

Comment:
We could, but we'd need to convert the embedded engines to strings later, otherwise we'd be generating atoms.